src/Extensions/Array.test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe } from 'riteway'
+import './Array'
+
+describe('Array', async (should: any) => {
```

Comment:
What do you think about explicitely grouping tests for this function?

`describe('Array.toObject', ...`

Reply:
yeah, sounds good!